chat-model/src/main/resources/canon/chat.json

```diff
@@ -1261,6 +1261,29 @@
           }
         }
       },
+      "ReactionDetailsPayload": {
+        "type": "object",
+        "extends": "#/components/schemas/MaestroPayload",
+        "properties": {
+          "action": {
+            "type": "string"
+          },
+          "reactionName": {
+            "type": "string"
```

Comment:
Is this free form or would an enum be appropriate?

Reply:
We would prefer not to define an enum for this too, as the accepted values will be extended in the future release.